bugbot/rules/crash_small_volume.py

```diff
@@ -181,6 +181,18 @@ def set_autofix(self, bugs):
                 )
                 autofix["keywords"] = {"remove": list(bug["keywords_to_remove"])}
 
+                # Clear needinfo flags requested by BugBot relating to increasing severity
+                if "topcrash" in bug["keywords_to_remove"]:
+                    flag_id = self.get_needinfo_topcrash_ids
+                    if flag_id is not None:
```

Comment:
why do we need this?

Reply:
Forgot to remove it, should be removed now!